containers/zammad/docker-entrypoint.sh

```diff
@@ -121,7 +121,8 @@ if [ "$1" = 'zammad-nginx' ]; then
   check_zammad_ready
 
   # configure nginx
-  sed -e "s#listen .*#listen ${NGINX_PORT};#g" \
+  sed -e "/\[..\]:80/d" \
+      -e "s#listen .*#listen ${NGINX_PORT};#g" \
```

Comment:
The new regex is superfluous. We can do this in the existing line, using the same ipv6 config in the container as well:

```
s#\(listen\)\(.*\)80#\1\2${NGINX_PORT}#g"
```